modules/secret/main.tf

```diff
@@ -39,49 +39,3 @@ resource "google_secret_manager_secret_iam_binding" "authorize-version-adder" {
 // Get a project number for this project ID.
 data "google_project" "project" { project_id = var.project_id }
 
-// Create an alert policy to notify if the secret is accessed by an unauthorized entity.
```

Comment:
Are there any we would consider keeping? If so, secret access seems like a candidate.

Reply:
agree this might have been one that matt wanted to keep

Reply:
many of the "secrets" are in fact ConfigMap so probably not work keeping them as libraries. if we want to keep them I'd say we have to be explicit in an "alerting" module instead of a resource module where people may just accidentally create alerts in their dev env without knowing why

Reply:
what about an opt in var to the sercrets module?

Reply:
I reverted this module